applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -983,13 +983,24 @@ static void agps_request_handle(struct nrf_modem_gnss_agps_data_frame *incoming_
 	 * to send the request.
 	 */
 #if defined(CONFIG_NRF_CLOUD_AGPS) && defined(CONFIG_NRF_CLOUD_MQTT)
+	uint32_t sv_mask_ephe = incoming_request->sv_mask_ephe;
+
+#if defined(CONFIG_NRF_CLOUD_PGPS)
+	incoming_request->sv_mask_ephe = 0;
```

Comment:
When both A-GPS and P-GPS are used, should also the request for almanacs be ignored? That's how it's implemented at least in the GNSS sample. If device has valid ephemerides all the time, there should be no need to download almanacs.

Reply:
Good point.